frontend/src/hooks/useUrlFilters.js

```diff
@@ -11,18 +12,36 @@ const { history } = window;
  * @param {Object[]} defaultFilters
  * @returns {[ Object[], Function ]}
  */
-export default function useUrlFilters(defaultFilters) {
+export default function useUrlFilters(key, defaultFilters) {
```

Comment:
I'm going to preface this suggestion by saying it may not be a good one, but it seems like this hook is no longer the `useUrlFilters` hook but `useSessionFiltersAndReflectInUrl` hook. Can we split these loading/reflecting filters in the URL and storing/loading filters from session storage into separate hooks and call one from the other?